fastlane/Fastfile

```diff
@@ -31,15 +31,21 @@ platform :android do
   end
   lane :playstore do
     sh("cd ..;bash scripts/build_android.sh")
-    
-    gradle(
-      task: 'assemble',
-      build_type: 'Release'
-    )
     isOnMasterBranch = currentBranch() == "master"
     if isOnMasterBranch
-      upload_to_play_store( track: 'beta', 
-      						json_key: 'key.json', 
+      gradle(
+        task: 'assemble',
+        build_type: 'Release'
+      )
+    else
+      gradle(
+        task: 'assemble',
+        build_type: 'nonmaster'
```

Comment:
Why not just usw Debug?

Reply:
As there is no "Debug" target there, and it's not a "Debug" build we're doing here. It's just not signing the result as we don't have a proper key.

But I'm not at all the Android / Fastlane / Gradle... expert to really know what's going on here. All I want to do is to fix the annoying CI build fail.